xdsl/transforms/convert_linalg_to_loops.py

```diff
@@ -0,0 +1,152 @@
+from collections.abc import Sequence
+from itertools import compress
+
+from xdsl.dialects import affine, arith, linalg, memref, scf
+from xdsl.dialects.builtin import (
+    AffineMapAttr,
+    IndexType,
+    IntegerAttr,
+    ModuleOp,
+)
+from xdsl.ir import Block, BlockArgument, MLContext, Operation, Region, SSAValue
+from xdsl.ir.affine import AffineDimExpr, AffineMap
+from xdsl.passes import ModulePass
+from xdsl.pattern_rewriter import (
+    GreedyRewritePatternApplier,
+    PatternRewriter,
+    PatternRewriteWalker,
+    RewritePattern,
+    op_type_rewrite_pattern,
+)
+
+
+def indices_for_map(
```

Comment:
What does this method do? Some documentation would help; specifically on the `else` bit where unused dimensions are filtered out. I guess this dictates if a load/store ends up being indexed (i.e., being dependent on) by one of the resulting loops.